Nikita_Konkov_Application/CSharpFactoryMethod/CSharpFactoryMethod/Developers/IDeveloper.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace CSharpFactoryMethod
+{
+    public interface IDeveloper
+    {
+        public virtual object Create(){ return new Object(); }
```

Comment:
no need to implement this method in Interface. And return type should be IVehicle.